lib/saml/url_service.rb

```diff
@@ -33,6 +33,13 @@ def mfa_url(current_user, success_relay: nil)
       build_sso_url(authn_context: authn_context, connect: policy, success_relay: success_relay)
     end
 
+    # This is the internal vets-api url that first gets invoked, it should redirect without authentication
+    # when this url gets invoked, the session should be destroyed, before the callback returns
+    def logout_url(session)
+      token = Base64.urlsafe_encode64(session.token)
+      Rails.application.routes.url_helpers.logout_v0_sessions_url(method: 'delete', session: token)
```

Comment:
Similarly, the route for this is defined as a GET action, but you're returning it as a DELETE here.

Reply:
Where am I returning it as a delete? We're doing a redirect.

Reply:
If the session lookup fails because its an invalid cookie, it will result in a 500 error probably, but that could be handled separately, maybe some instrumentation would be good around that. But the response is always going to be a redirect.

Reply:
In this line, you have a `method: 'delete'`, unless I'm misunderstanding what that is supposed to be for.